pom.xml

```diff
@@ -43,7 +43,7 @@
         <!-- Dependency versions -->
         <slf4j.version>1.7.32</slf4j.version>
         <netty.version>4.1.73.Final</netty.version>
-        <netty.tcnative.version>2.0.48.Final</netty.tcnative.version>
+        <netty.tcnative.version>2.0.46.Final</netty.tcnative.version>
```

Comment:
As discussed only, hope you have checked my `mvn dependency:tree` that grpc version aligns with netty version and doesn't bring different tcnative version

Reply:
Yeah, I have tried that. Also, I have verified by deploying a 3 cluster node with these changes.

Reply:
Command used: `mvn dependency:tree -Dincludes=io.grpc:grpc-netty-shaded`